#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raises a NameError for exception with message."""
    raise NameError(message)

